packages/eslint-plugin/tests/rules/no-enum-literals.test.ts

```diff
@@ -0,0 +1,142 @@
+import rule from '../../src/rules/no-enum-literals';
+import { RuleTester } from '../RuleTester';
+
+const ruleTester = new RuleTester({
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    project: './tsconfig.json',
+  },
+});
+
+ruleTester.run('no-enum-literals', rule, {
+  valid: [
+    '0 === 0',
```

Comment:
please add a comment explaining why you're testing non enum checks in an enum rule.

Reply:
I was guarding against false negatives. Added comment